tests/client/SwaggerClient/getattr_test.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 import pytest
 
-from bravado.client import ResourceDecorator
+from bravado.client import ResourceDecorator, SwaggerClient
```

Comment:
Please install the pre-commit hooks (`tox -e pre-commit install`) and run them (`.tox/pre-commit/bin/pre-commit run --all-files`). This will happen automatically for the files you changed for every git commit after you installed them.

Reply:
Done.